File: teti_Rangel/Triangulo.py
import math

class Triangulo:
    def __init__(self):
        self.ladoA = 0
        self.ladoB = 0
        self.ladoC = 0
        
    def getPerimetro(self):
        return self.ladoA+self.ladoB+self.ladoC
        
    
    def getMaiorLado(self):
        return  max(self.ladoA, self.ladoB, self.ladoC)
    def getArea(self):
        return self.ladoA*self.ladoB*self.ladoC
        # perimetro = int(self.getPerimetro/2)
        # p = perimetro
        # A   = self.ladoA
        # B  = self.ladoB
        # C   = self.ladoC
        # area = math.sqrt(p*(p-A)*(p-B)*(p-C))
        # return float(f"{area:2f}")
    

    
        
        

File: teti_Rangel/static/js/triangulo.js
const ladoA = document.getElementById('ladoA');
const ladoB = document.getElementById('ladoB');
const ladoC = document.getElementById('ladoC');
const area = document.getElementById('area');
const maiorLado = document.getElementById('maiorLado');
const perimetro = document.getElementById('perimetro');
const trianguloParaCima = document.getElementById('triangulo-para-cima');

trianguloParaCima.style.borderLeftWidth = `${ladoA.value}px` 
trianguloParaCima.style.borderRightWidth = `${ladoB.value}px`
trianguloParaCima.style.borderBottomWidth = `${ladoC.value}px`

function mudarBorder(e){
    trianguloParaCima.style.borderLeftWidth = `${ladoA.value}px` 
    trianguloParaCima.style.borderRightWidth = `${ladoB.value}px`
    trianguloParaCima.style.borderBottomWidth = `${ladoC.value}px`
}
function maiorLadoF(){
    maiorLado.style.visibility = 'visible'
    perimetro.style.visibility = 'hidden'
    area.style.visibility = 'hidden'
}
function perimetroF(){
    perimetro.style.visibility = 'visible'
    maiorLado.style.visibility = 'hidden'
    area.style.visibility = 'hidden'
}

function areaF(){
    area.style.visibility = 'visible'
    maiorLado.style.visibility = 'hidden'
    perimetro.style.visibility = 'hidden'
}
function enviarDados(e){
    e.preventDefault()
    return false
}
